### PR TITLE
[TASK-7] [Test] As a user, I can load more or refresh surveys list

### PR DESCRIPTION
**Pull request by minhnimble**
https://github.com/minhnimble/android-surveys-list/projects/1#card-46082926

## What happened

Need to write unit tests for classes used in pagination and refreshing surveys list feature.
 
## Insight

**Acceptance Criteria**
- [x] Write unit tests
 
## Proof Of Work
No visible changes, just under the hood tests 🙏

### CODE REVIEW

**Review by manh-t — APPROVED** [review]

File: app/src/test/java/co/nimblehq/usecase/survey/DeleteLocalSurveysExcludeIdsCompletableUseCaseTest.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package co.nimblehq.usecase.survey
+
+import co.nimblehq.data.lib.schedulers.TestRxSchedulerProviderImpl
+import co.nimblehq.data.storage.dao.SurveyDao
+import com.nhaarman.mockitokotlin2.mock
+import org.junit.Before
+import org.junit.Test
+
+class DeleteLocalSurveysExcludeIdsCompletableUseCaseTest {
```
Comment (hoangnguyen92dn) [inline app/src/test/java/co/nimblehq/usecase/survey/DeleteLocalSurveysExcludeIdsCompletableUseCaseTest.kt:9]: Could you please add the test for error case?

File: app/src/test/java/co/nimblehq/ui/screen/main/surveys/SurveysViewModelTest.kt [hunk 2]
```diff
@@ -110,4 +112,197 @@ class SurveysViewModelTest {
...
+
+        // Assert
+        selectedSurveyIndexObserver
+            .assertNoErrors()
+            .assertValueCount(2)
+            .assertValueAt(1) { it == 1 }
+    }
+
+    @Test
+    fun `When user swipe to next page and the new value is invalid, it doesn't update anything`() {
```
Comment (hoangnguyen92dn) [inline app/src/test/java/co/nimblehq/ui/screen/main/surveys/SurveysViewModelTest.kt:281]: Optional: We could verify `NoMoreSurveysError` in this testcase for coverage improvement.

File: app/src/test/java/co/nimblehq/usecase/survey/DeleteLocalSurveysExcludeIdsCompletableUseCaseTest.kt (hunk 1 above)
Comment (minhnimble) [reply app/src/test/java/co/nimblehq/usecase/survey/DeleteLocalSurveysExcludeIdsCompletableUseCaseTest.kt:9]: yes sure, will add 👍, knowing how to do it now 😁

File: app/src/test/java/co/nimblehq/ui/screen/main/surveys/SurveysViewModelTest.kt (hunk 2 above)
Comment (minhnimble) [reply app/src/test/java/co/nimblehq/ui/screen/main/surveys/SurveysViewModelTest.kt]: sure, let me update update

**Review by hoangnguyen92dn — APPROVED** [review]

**Pull request merged**